android/BOINC/app/src/main/java/edu/berkeley/boinc/client/NoticeNotification.java

```diff
@@ -172,27 +164,16 @@ private Notification buildNotification() {
     private static final Bitmap getLargeProjectIcon(final Context context, final String projectName) {
         final Bitmap projectIconBitmap;
         try {
-            return (projectIconBitmap = Monitor.getClientStatus().getProjectIconByName(projectName)) != null ?
-                    Bitmap.createScaledBitmap(
-                            projectIconBitmap,
-                            projectIconBitmap.getWidth() << 1,
-                            projectIconBitmap.getHeight() << 1,
-                            false
-                    ) :
-                    BitmapFactory.decodeResource(
-                            context.getResources(),
-                            R.drawable.ic_stat_notify_boinc_normal
-                    );
-        } catch (Exception e) {
-            if (Log.isLoggable(Logging.TAG, Log.DEBUG)) Log.d(
-                    Logging.TAG,
-                    e.getLocalizedMessage(),
-                    e
-            );
-            return BitmapFactory.decodeResource(
-                    context.getResources(),
-                    R.drawable.ic_stat_notify_boinc_normal
-            );
+            return (projectIconBitmap = Monitor.getClientStatus().getProjectIconByName(projectName)) !=
```

Comment:
This whole block is a type of change that makes things worse. The human editor had clear and justified intentions (with the line breaks) which should be preserved.